vendor/assets/javascripts/react-chosen.js

```diff
@@ -0,0 +1,69 @@
+(function(window, React, $) {
```

Comment:
We ran into the issue that both `rails-assets-react-chosen` and `react-rails` include `react.js`, but they aren't the same versions.  Sprockets ends up just picking one of the two, and if it picks the `react-assets-react-chosen` one our code breaks.

It's maybe possible to exclude certain `rails-assets` dependencies from the assets path—don't remember if we tried that.